pkg/webui/console/components/lorawan-version-input/index.js

```diff
@@ -49,6 +53,10 @@ const phyVersionsMap = {
   [PHY_V1_0_3_REV_A.value]: [MAC_V1_0_3, MAC_V1_0_4],
   [PHY_V1_1_REV_A.value]: [MAC_V1_1, MAC_V1_0_4],
   [PHY_V1_1_REV_B.value]: [MAC_V1_1, MAC_V1_0_4],
+  [RP002_V1_0_0.value]: [MAC_V1_1, MAC_V1_0_4],
+  [RP002_V1_0_1.value]: [MAC_V1_1, MAC_V1_0_4],
+  [RP002_V1_0_2.value]: [MAC_V1_1, MAC_V1_0_4],
+  [RP002_V1_0_3.value]: [MAC_V1_1, MAC_V1_0_4],
```

Comment:
Actually I believe that RP2 can be used with all LoRaWAN versions, right @adriansmares ?

Reply:
In theory yes. 

In practice I haven't seen any stack that backports the versions - I see 1.0.4+1.1 RP2 based releases in practice.

We could show them everywhere, but I'm not sure if its worth confusing the user with more options. I'd say that we add them on customer demand.

Reply:
So I leave it as it is now?

Reply:
Yes.